src/app/components/LinkedData/index.tsx

```diff
@@ -111,12 +111,37 @@ const LinkedData = ({
 
   const hasByline = !!bylineLinkedData;
 
-  const { authorName, authorTopicUrl, twitterLink, authorImage, location } =
-    bylineLinkedData || {};
+  const places: string[] = [];
 
-  const sameAs = [authorTopicUrl, twitterLink].filter(Boolean);
+  const getAuthorTagging = (data: BylineLinkedData | null) => {
+    const { authorName, authorTopicUrl, twitterLink, authorImage, location } =
+      data || {};
 
-  const locationCreated = { '@place': location };
+    const sameAs = [authorTopicUrl, twitterLink].filter(Boolean);
+    if (location) {
+      if (!places.includes(location)) {
+        places.push(location);
+      }
+    }
+
+    return {
+      '@type': 'Person',
+      name: authorName,
+      ...(sameAs.length && { sameAs }),
+      ...(authorImage && { image: authorImage }),
+    };
+  };
+  let bylineAuthors;
+  if (bylineLinkedData) {
+    bylineAuthors =
+      bylineLinkedData?.length > 1
+        ? bylineLinkedData?.map(data => getAuthorTagging(data))
+        : getAuthorTagging(bylineLinkedData?.[0]);
```

Comment:
map will still work if there's only 1 entry in the list

```suggestion
    const bylineAuthors = bylineLinkedData?.map(data => getAuthorTagging(data))
```

Reply:
Yeah, this is like this because if there's only one contributor the author value in the schema should be just one object as normal, rather than an array with one object in it. Maybe could check for this after this line though

Reply:
OK - perhaps move that logic to the bit where you're sending the byline authors to the schema?